kernel/src/main/java/org/kframework/EvaluationException.java

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2018 Runtime Verification, Inc. (RV-Match team). All Rights Reserved.
+package org.kframework;
+
+import java.util.ArrayList;
+import org.kframework.kore.K;
+
+
+public class EvaluationException extends RuntimeException {
+    private int steps_;
+    private ArrayList<K> stacktrace_;
+
+    public EvaluationException(int steps, ArrayList<K> stacktrace) {
+        super("EvaluationException");
```

Comment:
Should have a better message than this. Something that actually describes the error.

Reply:
What about `Execution got stuck while evaluating a K function.` ?